### DISCUSSION TITLE
Log & Log Entry record  not getting created.

### DISCUSSION THREAD

**Discussion by nihalsingh101** [jongpie/NebulaLogger#575] [General]
I've installed unmanaged package in sandbox, trying to execute the below line from developer console  anonymous window, platform event getting published(able to see in "Event Stream" tab) but Log & Log Entry record are not getting created. 

Logger.debug('Test Debug');
Logger.saveLog();

when trying to execute below line, getting exception.

Logger.info('hello').addTag('world');
Logger.saveLog(Logger.SaveMethod.SYNCHRONOUS_DML);

<img width="466" alt="image" src="https://github.com/jongpie/NebulaLogger/assets/108475847/eaa9f59d-ad76-4498-9cb2-edc3c640b7eb">

When trying this with manage package, all working expected.

thanks again in advance.

**Comment by jongpie** [comment]
Hi @nihalsingh101! Do you know which version of the unlocked package you're using? There was a bug in some versions when using a combination of `addTag()` and the save method `Logger.SaveMethod.SYNCHRONOUS_DML` that would prevent saving of data - the bug was fixed in [`v4.11.7`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.11.7). If you're on an older version of the unlocked package, I'd try upgrading to see if that solves the issue.

**Reply by nihalsingh101** [reply]
Hello @jongpie 
I'm using the **4.11.11**, do I need to replace **4.11.11** with [v4.11.7](https://github.com/jongpie/NebulaLogger/releases/tag/v4.11.7)?

Also when I'm trying to execute the below line, no error I'm getting but **Log & Log Entry records aren't getting created**. This is the also issue with version(**4.11.11**) that I'm using?

Logger.debug('Test Debug');
Logger.saveLog();

**Reply by jongpie** [reply]
Hi @nihalsingh101 - if you're already on `v4.11.11`, then I don't think it's the issue I previously mentioned (and you don't need to downgrade), it sounds like there's something else causing the issue. Based on your screenshot, it looks like there's an issue with the inner class `LogEntryEventHandler.StatusApiCalloutQueueable` that tries to make a callout to `api.status.salesforce.com`. I'm not sure why that specific line (687) would be throwing that exception, and it's possible that there's an issue earlier/elsewhere in Nebula Logger that's the root cause issue.

Could you try disabling the callout in your org to see if that prevents that exception? You can disable it by updating a custom metadata record, under Setup --> Custom Metadata Types --> Logger Parameters --> Call Status API. Just update the value (shown in the screenshot below) from 'true' to 'false'

![image](https://github.com/jongpie/NebulaLogger/assets/1267157/1c801654-8701-4a0d-8425-178f23f016c2)

Once you've update that record, try running this script to see if either of these Log Entry records are created:

```apex
// Normally, I would not recommend using this next line of code, but it's helpful for testing purposes
Logger.getUserSettings().LoggingLevel__c = LoggingLevel.INFO.name();

Logger.info('hello');
Logger.saveLog(Logger.SaveMethod.EVENT_BUS);
Logger.info('and hello again').addTag('world');
Logger.saveLog(Logger.SaveMethod.SYNCHRONOUS_DML);
```

**Reply by nihalsingh101** [reply]
@jongpie Thank you for your quick reply. It's working now.
